.github/workflows/frieeren-component-CD.yml

```
@@ -49,23 +64,30 @@ jobs:
         with:
           branch: ${{ github.ref }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          tags: true
 
       - name: Configure npm
         run: |
-          cat << EOF > "packages/frieeren-components/.npmrc"
-            email=frieeren2@gmail.com
+          cat << EOF > "${{ env.PROJECT_PATH }}/.npmrc"
+            email="${{ env.BOT_GITHUB_EMAIL }}"
             //registry.npmjs.org/:_authToken=$NPM_TOKEN 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)